CscopeSublime.sublime-settings

```diff
@@ -31,4 +31,8 @@
     // An optional custom command used to build cscope's database.
     // This must be in array format.
     //"database_build_command": [ "cscope-indexer", "-r" ],
+
+    // On default build command, whether to treat folders in project (after the 1st one) as
+    // additional source directories
+    "auto_next_source_dirs": false,
```

Comment:
I'm curious about the use case here. I've never been part of a project so big that it is split into multiple directories and is not contained within a common parent folder. Can you help me understand what kind of scenario would require this or how you have things set up such that this is useful?

Also, I wonder if we even need `auto_next_source_dirs` or if we can do the right thing automatically?

Reply:
Because of this reason I didn't want to make it default option :) In my case the other folders contain library-like project (used by other 'main' projects also) or headers of this library, it is handy to have these sources in the same project. Sublime Text automatically indexes all folders in the project for definitions, so I guess this is similar behavior.
Also, it wasn't possible to update or create database with more than 1 folder in project, because of restriction ```... len(project_info_paths) == 1```